### ISSUE TITLE
Tsup error transpiling on azure

### ISSUE THREAD

**Issue by blackmo18** [egoist/tsup#552]
Hi,

I just encountered an issue when deploying this to azure, as shown below.

I ran the same command in my local machine but can't replicate the issue.

any help pinpoint the root for this is greatly appreciated.

```
2022-01-28T15:53:17.407982992Z > tsup --minify client1.ts
2022-01-28T15:53:17.407991792Z
2022-01-28T15:53:17.569792965Z /node_modules/bundle-require/dist/index.js:418
2022-01-28T15:53:17.569827565Z   const preserveTemporaryFile = options.preserveTemporaryFile ?? !!process.env.BUNDLE_REQUIRE_PRESERVE;
2022-01-28T15:53:17.569834865Z                                                                ^
2022-01-28T15:53:17.569839665Z
2022-01-28T15:53:17.569844265Z SyntaxError: Unexpected token '?'
2022-01-28T15:53:17.569848865Z     at wrapSafe (internal/modules/cjs/loader.js:915:16)
2022-01-28T15:53:17.569853465Z     at Module._compile (internal/modules/cjs/loader.js:963:27)
2022-01-28T15:53:17.569858065Z     at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
2022-01-28T15:53:17.569878566Z     at Module.load (internal/modules/cjs/loader.js:863:32)
2022-01-28T15:53:17.569883666Z     at Function.Module._load (internal/modules/cjs/loader.js:708:14)
2022-01-28T15:53:17.569887866Z     at Module.require (internal/modules/cjs/loader.js:887:19)
2022-01-28T15:53:17.569891866Z     at require (internal/modules/cjs/helpers.js:74:18)
2022-01-28T15:53:17.569895966Z     at Object.<anonymous> (/node_modules/tsup/dist/chunk-XHRHUTCW.js:16:22)
2022-01-28T15:53:17.569900866Z     at Module._compile (internal/modules/cjs/loader.js:999:30)
2022-01-28T15:53:17.569904866Z     at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
2022-01-28T15:53:17.578424517Z npm info lifecycle ecomerce-js-extension@0.5.0~build_client_tsup: Failed to exec build_client_tsup script
2022-01-28T15:53:17.582907944Z npm ERR! code ELIFECYCLE
2022-01-28T15:53:17.582950344Z npm ERR! errno 1
2022-01-28T15:53:17.586467965Z npm ERR! my-sample-app@0.5.0 build_client_tsup: `tsup --minify client1.ts`
2022-01-28T15:53:17.586688467Z npm ERR! Exit status 1

```

**Comment by egoist** [comment]
Just fixed in 5.11.12

**Comment by blackmo18** [comment]
That was fast @egoist . Thanks
